src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```
@@ -67,13 +71,23 @@
-
+    isReadOnlyWithShowResult: boolean = false;
     // Icon
     farListAlt = faListAlt;
+    faPenSquare = faPenSquare;
+    faChevronDown = faChevronDown;
+    faCircleNotch = faCircleNotch;
+    faTimeline = faTimeline;
+    faEye = faEye;
+    participationUpdateListener: Subscription;
+    sortedHistoryResults: Result[];
+    hasAthenaResultForLatestSubmission: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)